sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOTest.java

```diff
@@ -466,40 +500,9 @@ public void testCustomMapperImplDelete() {
     assertEquals(1, counter.intValue());
   }
 
-  @Test
-  public void testSplit() throws Exception {
-    PipelineOptions options = PipelineOptionsFactory.create();
```

Comment:
Once we have the SplitFn we should probably test this, actually we had a lot of historic issues with Splitting on this connector so please pay attention to ensure we don't introduce regressions.

Reply:
Hmm, if you know of relevant issues that we previously had, lmk, I will also look through the issue tracker to see if there were previous bugs.